BrowserKit/Sources/Common/Constants/StandardImageIdentifiers.swift

```diff
@@ -69,6 +69,7 @@ public struct StandardImageIdentifiers {
         public static let deviceTablet = "deviceTabletLarge"
         public static let download = "downloadLarge"
         public static let edit = "editLarge"
+        public static let ellipsis = "ellipsisLarge"
```

Comment:
@thatswinnie Do you know if we can we add haddock icons, or do we need to go through a process with design system? Asking since I remember you worked on the image identifiers at one point

Reply:
@adudenamedruby do you have this answer maybe?

Reply:
what's haddock icons? This seems fine to me as it's from the new acorn system